drivers/timer/hpet.c

```diff
@@ -604,11 +598,11 @@ int _sys_clock_driver_init(struct device *device)
 
 	counter_load_value = (u32_t)(tickFempto / hpetClockPeriod);
 
-	DBG("\n\nHPET: configuration: 0x%x, clock period: 0x%x (%d pico-s)\n",
+	SYS_LOG_DBG("HPET: configuration: 0x%x, clock period: 0x%x (%d pico-s)",
```

Comment:
Log messages include the current function name so consider dropping the "HPET: " prefix.